src/pages/App.jsx

```diff
@@ -62,6 +63,7 @@ const App = (props) => {
             </Task>
           );
         })}
+        <Toast iconName="check">The task has been successfully created</Toast>
```

Comment:
I don't think it is bad that the `iconName` is customizable, but I feel you will always set it to `check` in the Success toasts (the only one we have created as of now)

I wouldn't send it as prop until there is a need for `errors` or `warning` toasts. Even when that happens there may be better ways to manage it

Reply:
it's kind of weird you can customize the icon and not the background-color don't you think?